Detectors/TOF/workflow/src/TOFRawWriterSpec.cxx

```diff
@@ -50,14 +50,18 @@ void RawWriter::run(ProcessingContext& pc)
 
   int nwindowperorbit = Geo::NWINDOW_IN_ORBIT;
   int nwindowintimeframe = 256 * nwindowperorbit;
```

Comment:
here too, hardcoded 256 HBs, should be a parameter.

Reply:
@noferini , is this also resolved by your PR?